app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -32,17 +32,17 @@ expander:
     text: "You can normally only do a teacher degree apprenticeship (TDA) if you've been a resident in the UK for the last 3 years or more. There are some exceptions to this. For example, if you've applied to the Afghan or Ukraine resettlement schemes. Speak to providers for more information."    
 ---
 
-## Undergraduate degree courses
-You need a bachelor’s degree in any subject to train to teach in primary, secondary and special schools in England. If you do not already have a degree, you can train to be a teacher as part of your bachelor’s degree and get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
+## Bachelor's degree courses
+You need a bachelor’s degree in any subject to train to teach in primary, secondary and special schools in England. If you do not already have a bachelor's degree, you can train to be a teacher as part of your bachelor’s degree and get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
```

Comment:
```suggestion
You need a bachelor’s degree in any subject to train to teach in primary, secondary and special schools in England. If you do not already have one, you can train to be a teacher as part of a bachelor’s degree and get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
```
Is it worth taking out one of the `bachelor's` references - it is quite repetitive? But disagree if needed!